ui/app/styles/components/http-requests-bar-chart.scss

```diff
@@ -37,3 +37,30 @@
     }
   }
 }
+
+.d3-tooltip {
+  line-height: 1.25;
+  padding: $spacing-s;
+  background: $grey;
+  color: $ui-gray-010;
+  border-radius: 2px;
+}
+
+/* Creates a small triangle extender for the tooltip */
+.d3-tooltip:after {
```

Comment:
Pseudo element selectors 'should be' double colon `::`

https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements

> As a rule, double colons (::) should be used instead of a single colon (:). This distinguishes pseudo-classes from pseudo-elements. However, since this distinction was not present in older versions of the W3C spec, most browsers support both syntaxes for the original pseudo-elements.

i.e. Both work but depends how 'exact' you want to be.